src/types/colorTable.cpp

```diff
@@ -286,85 +286,123 @@ static constexpr std::array<til::color, 256> standardXterm256ColorTable{
     til::color{ 0xEE, 0xEE, 0xEE },
 };
 
+// Routine Description:
+// - Calculate XOrg app color variant with given base color and variant index.
+//   See also https://en.wikipedia.org/wiki/X11_color_names
+// Arguments:
+// - baseColor - The base color to use.
+// - variant - The variant index.
+// Return Value:
+// - The result color
+static constexpr til::color _calculateXorgAppColorVariant(til::color baseColor, size_t variant)
+{
+    float ratio = 1.0f;
+    switch (variant)
+    {
+    case 1:
+        ratio = 1.0f;
+        break;
+    case 2:
+        ratio = 0.932f;
+        break;
+    case 3:
+        ratio = 0.804f;
+        break;
+    case 4:
+        ratio = 0.548f;
+        break;
+    default:
+        break;
+    }
+
+    return til::color(
+        ::base::saturated_cast<uint8_t>(baseColor.r * ratio),
+        ::base::saturated_cast<uint8_t>(baseColor.g * ratio),
+        ::base::saturated_cast<uint8_t>(baseColor.b * ratio));
+}
+
+static constexpr til::presorted_static_map xorgAppStandardVariantColorTable{
+    std::pair{ "antiquewhite"sv, std::array<til::color, 2>{ til::color{ 250, 235, 215 }, til::color{ 255, 239, 219 } } },
+    std::pair{ "brown"sv, std::array<til::color, 2>{ til::color{ 165, 42, 42 }, til::color{ 255, 64, 64 } } },
+    std::pair{ "burlywood"sv, std::array<til::color, 2>{ til::color{ 222, 184, 135 }, til::color{ 255, 211, 155 } } },
+    std::pair{ "cadetblue"sv, std::array<til::color, 2>{ til::color{ 95, 158, 160 }, til::color{ 152, 245, 255 } } },
+    std::pair{ "chocolate"sv, std::array<til::color, 2>{ til::color{ 210, 105, 30 }, til::color{ 255, 127, 36 } } },
+    std::pair{ "coral"sv, std::array<til::color, 2>{ til::color{ 255, 127, 80 }, til::color{ 255, 114, 86 } } },
+    std::pair{ "darkgoldenrod"sv, std::array<til::color, 2>{ til::color{ 184, 134, 11 }, til::color{ 255, 185, 15 } } },
+    std::pair{ "darkolivegreen"sv, std::array<til::color, 2>{ til::color{ 85, 107, 47 } } },

```

Comment:
In the xorgAppStandardVariantColorTable table, why do darkolivegreen and turquoise only have one item where as all the others have two?

Reply:
Sharp eyes. That is a mistake. Thanks for catching this.

Huh, I thought compiler would warn me about some things. Say I have a `std::array<x, 2>` and I put 3 objects in it, the compiler would warn me. But it does not care if the elements are not enough?